src/main/modules/ardoq/ArdoqClient.ts

```diff
@@ -150,13 +153,17 @@ export class ArdoqClient {
     }
   }
 
-  public async updateDep(d: Dependency): Promise<[ArdoqComponentCreatedStatus, string | null]> {
+  public async updateDep(
+    d: Dependency,
```

Comment:
now your second argument has a long name having this just called d looks a bit odd